apps/src/aichat/constants.ts

```diff
@@ -2,3 +2,5 @@ import {ModelDescription} from './types';
 import modelsJson from '@cdo/static/aichat/modelDescriptions.json';
 
 export const modelDescriptions: ModelDescription[] = modelsJson;
+
+export const ANALYTICS_SAMPLE_RATE = 0.1;
```

Comment:
From [spec](https://docs.google.com/document/d/1yIez6ooHbfwfM2W16pQvGHgKTe97AdGJN7YECTOpmBU/edit#heading=h.ghfocbs7b58g), we want to consider sampling (1/100 or 1/1000).

Reply:
Checked with Sam and she said let's hold off on sampling for now. Commit that removes sampling: https://github.com/code-dot-org/code-dot-org/pull/58397/commits/342575cf0d9099980d740ee76008e24f7d48f47f